src/intents.cs

```diff
@@ -971,7 +971,7 @@ enum INWorkoutNameIdentifier {
 	[Introduced (PlatformName.iOS, 10, 0)]
 	[Unavailable (PlatformName.MacOSX)]
 	[Internal]
-	[Category]
+	[Category (ignoreStaticWarnings: true)]
```

Comment:
same here, but it's the type (not the method) that is internal - both should be checked